request_test.go

```diff
@@ -231,6 +232,7 @@ func TestRequestsAsync(t *testing.T) {
 		{req: NewRollbackRequest(), async: false},
 		{req: NewIdRequest(validProtocolInfo), async: false},
 		{req: NewBroadcastRequest(validKey), async: false},
+		{req: NewWatchOnceRequest(validKey), async: false},
```

Comment:
We need also tests for:

1. https://github.com/tarantool/go-tarantool/blob/7320b057d196f7c167d7ea197a192a76d760dda7/request_test.go#L245
2. https://github.com/tarantool/go-tarantool/blob/7320b057d196f7c167d7ea197a192a76d760dda7/request_test.go#L279
3. test for the body encoding (see for other requests in the file).